shell/src/main/java/tachyon/shell/command/RmrCommand.java

```diff
@@ -46,6 +47,8 @@ void runCommand(TachyonURI path) throws IOException {
       TachyonFile fd = mTfs.open(path);
       mTfs.delete(fd, options);
       System.out.println(path + " has been removed");
+    } catch (InvalidPathException e) {
+      System.out.println(path + " does not exist");
```

Comment:
@uronce-cc I don't think it's a good idea to print out the exception in tfs shell. We should make the tfs shell message to be as much concise and clean as possible, and currently other commands do not print out exception messages as well.